lib/jekyll-sitemap.rb

```diff
@@ -59,7 +59,7 @@ def sitemap_content
       site_map.content = File.read(source_path)
       site_map.data["layout"] = nil
       site_map.render({}, @site.site_payload)
-      site_map.output.gsub(/\s*\n+/, "\n")
+      site_map.output.gsub(/\s{2,}/, "\n")
```

Comment:
Yes, this was intended.
`sitemap.xml` is only intended to be read by machines, so indentation is irrelevant.